sample_apps/run_example.sh

```diff
@@ -582,6 +686,26 @@ function construct_policyKey_platform_is_trusted_simple_app_under_oe() {
     run_popd
 }
 
+# ###########################################################################
+function construct_policyKey_platform_is_trusted_simple_app_under_gramine() {
+    run_cmd
+    run_pushd "${PROV_DIR}"
+
+    # The docs state that this certificate should exist at a known location.
```

Comment:
A big NOTE has been added to [instructions.md](https://github.com/vmware-research/certifier-framework-for-confidential-computing/blob/agurajada/71-run_example_for-gramine-app/sample_apps/simple_app_under_gramine/instructions.md) right after the **Obtain Intel Certificates** section.